src/stk/molecular/topology_graphs/polymer/linear/linear.py

```diff
@@ -245,6 +261,12 @@ def __init__(
             length of `orientations` must be equal to
             ``len(repeating_unit)*num_repeating_units``.
 
+            If there is only one building block in the constructed
+            polymer i.e. the `repeating_unit` has a length of 1 and
```

Comment:
this is a little unclear, should be ."i.e. there is only one building block, the `repeating_unit` ....."